### PR TITLE
test(Communities) Adding permission test automated

### PR DESCRIPTION
**Pull request by Valentina1133**
Added new test for adding permissions in communities
https://github.com/status-im/desktop-qa-automation/issues/89

### CODE REVIEW

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 1]
```diff
@@ -568,4 +568,90 @@ def verify_action_button_present(self, option:str, enabled: bool):
...
+        raise LookupError('Add button for allowed to not found ')
+
+    def open_in_context_menu(self):
+        for child in walk_children(self._in_list_item.object):
+            if getattr(child, 'id', '') == 'addItemButton':
+                click_obj(child)
+                return
+        raise LookupError('Add button for in not found ')
+
+    def check_tags_titles(self, title: str, second_title: str, third_title: str):
```
Comment (StateOf-Vlado) [inline test/ui-test/src/screens/StatusCommunityScreen.py:647]: let's make it as a property:
```
@property
def tags() -> typing.List[str]:
tags_titles = []
for child in walk_children(self._tag_item.object):
           if getattr(child, 'title', '') in checked_titles_list:
                tags_titles.append(str(child.title))
return tags_titles
```

verification should in step

File: test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py [hunk 2]
```diff
@@ -170,16 +172,29 @@ def step(context, community_sidebar_option:str):
...
+    permission_view.set_in_community(in_general)
+    permission_view.create_permission()
 
 #########################
 ### VERIFICATIONS region:
 #########################
 
+@Then("created permission with \"|any|\" and \"|any|\" and \"|any|\" is on permission page")
+def step(context, title, second_title, third_title):
+    _permissionsView.check_tags_titles(title, second_title, third_title)
```
Comment (StateOf-Vlado) [inline test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py:196]:
```
tags = _permissionsView.tags
if title != 'No':
assert title in tags
if second_title != 'No':
assert title in tags
if second_title != 'No':
assert third_title in tags
```

File: test/ui-test/src/screens/StatusCommunityScreen.py (hunk 1 above)
Comment (Valentina1133) [reply test/ui-test/src/screens/StatusCommunityScreen.py]: Done

File: test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py (hunk 2 above)
Comment (Valentina1133) [reply test/ui-test/testSuites/suite_communities/shared/steps/communitySteps.py]: Done

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 3]
```diff
@@ -568,4 +567,83 @@ def verify_action_button_present(self, option:str, enabled: bool):
...
+        raise LookupError('Add button for allowed to not found ')
+
+    def open_in_context_menu(self):
+        for child in walk_children(self._in_list_item.object):
+            if getattr(child, 'id', '') == 'addItemButton':
+                click_obj(child)
+                return
+        raise LookupError('Add button for in not found ')
+
+    @property
```
Comment (StateOf-Vlado) [inline test/ui-test/src/screens/StatusCommunityScreen.py:646]: Please move it to the top of class
https://stackoverflow.com/questions/10289461/what-is-a-good-way-to-order-methods-in-a-python-class

**Review by StateOf-Vlado — APPROVED** [review]
It's good to have a description in PR, at least a link to the task
like this:
image

File: test/ui-test/src/screens/StatusCommunityScreen.py (hunk 3 above)
Comment (Valentina1133) [reply test/ui-test/src/screens/StatusCommunityScreen.py]: Done + replaced with method from SquishDriver

**Pull request merged**